x-pack/test/functional_with_es_ssl/apps/ml/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ loadTestFile, getService }: FtrProviderContext) => {
+  const ml = getService('ml');
+
+  describe('ML app', function () {
+    this.tags(['mlqa', 'skipFirefox']);
+
+    before(async () => {
+      await ml.securityCommon.createMlRoles();
+      await ml.securityCommon.createMlUsers();
+    });
+
+    after(async () => {
```

Comment:
Since we're loading a dataset and creating an index pattern in the test file, we should make sure that we leave nothing behind by adding some steps in this `after` method:
```
        await ml.testResources.deleteIndexPatternByTitle('ft_ecommerce');
        await esArchiver.unload('ml/ecommerce');
```

Reply:
Added in https://github.com/elastic/kibana/pull/89286/commits/3e569c79418823882d7892129037866ac5c1bf0c